website/docs/docs/dbt-cloud/on-premises/dependencies.md

```diff
@@ -65,15 +65,15 @@ In order to perform basic maintenance and license checking, the following outbou
 
 - `api.replicated.com`: This endpoint services the license sync check and is used to pull down yaml files for app upgrades.
 
-### Optional External Dependencies
+## Optional External Dependencies
 
-#### Auth0
+### Auth0
```

Comment:
fyi our new sso integrations do not use Auth0 - Cloud will negotiate SSO with the identity providers natively. I think we can either remove this, or just note that Auth0 is "legacy" and no longer required for new deployments?

Reply:
I removed this section for now. I think we should add a section on Okta/GSuite (as there is config for on-prem in the admin console for those). I made a ticket for myself to add that in the next PR with the on-prem setup updates.